code/datums/Emote_system/emotes.dm

```diff
@@ -0,0 +1,1726 @@
+	if(isalienadult(user))
+		return 1
+
+// sorry, no selftext for humans as we'd have to add a var to species for it and that's yuk - VB
+/datum/emote/deathgasp/human
+	text = "dummytext"
+
+/datum/emote/deathgasp/human/available(var/mob/user)
+	if(ishuman(user))
+		return 1
+
+/datum/emote/deathgasp/human/standardMessage(var/mob/user, var/list/params)
+	var/mob/living/carbon/human/U = user
+	var/message = "<span class = '[userSpanClass]'>\The [user]</span> [U.species.death_message]"
+	return message
+
+/datum/emote/deathgasp/human/createSelfMessage(var/mob/user, var/list/params, var/message)
+	return message
+
+/datum/emote/deathgasp/human/replaceMobWithYou(var/mob/user, var/message)
+	return message
+
+/datum/emote/deathgasp/robot
+	text = "shudders violently for a moment, then becomes motionless, its eyes slowly darkening"
+	selfText = "shudder violently for a moment, then become motionless, your eyes slowly darkening"
+	audible = 0
+
+/datum/emote/deathgasp/robot/available(var/mob/user)
+	if(isrobot(user))
+		return 1
+
+/datum/emote/drone
+	name = "drone"
+	desc = "Makes the mob drone and rumble"
+	commands = list("drone", "drones", "rumble", "rumbles", "hum", "hums")
+	text = "rumbles"
+	selfText = "rumble"
+	audible = 1
+	sound = 'sound/voice/DraskTalk.ogg'
+	canTarget = 1
+	targetMob = 1
+
+/datum/emote/drone/available(var/mob/user)
+	if(user.get_species() == "Drask")
+		return 1
+
+/datum/emote/drone/addTarget(var/mob/user, var/list/params, var/message)
+	if(!params["target"])
+		return message
+	message = replacetext(message, "rumbles", "drones")
+	return ..()
+
+/datum/emote/drone/createSelfMessage(var/mob/user, var/list/params, var/message)
+	message = ..()
+	message = replacetext(message, "drones", "drone")
+	return message
+
+/datum/emote/drool
+	name = "drool"
+	desc = "Makes the mob drool"
+	commands = list("drool", "drools")
+	text = "drools"
+	selfText = "drool"
+
+/datum/emote/drool/available(var/mob/user)
+	if(isalienadult(user) || islarva(user))
+		return 1
+	if(ishuman(user))
+		return 1
+
+/datum/emote/eyebrow
+	name = "eyebrow"
+	desc = "Makes the mob raise an eyebrow"
+	commands = list("eyebrow")
+	text = "raises an eyebrow"
+	selfText = "raise an eyebrow"
+
+/datum/emote/eyebrow/available(var/mob/user)
+	if(ishuman(user))
+		return 1
+
+/datum/emote/faint
+	name = "faint"
+	desc = "makes the mob faint"
+	commands = list("faint", "faints")
+	text = "faints"
+	selfText = "faint"
+
+/datum/emote/faint/available(var/mob/user)
+	if(ishuman(user))
+		return 1
+
+/datum/emote/faint/doAction(var/mob/user, var/list/params)
+	if(!user.sleeping)
+		user.sleeping += 1
+
+/datum/emote/fart
+	name = "fart"
+	desc = "makes the mob fart"
+	commands = list("fart", "farts")
+	text = "farts"
+	selfText = "fart"
+	cooldown = 50
+	audible = 1
+
+/datum/emote/fart/available(var/mob/user)
+	if(ishuman(user))
+		return 1
+
+/datum/emote/fart/standardMessage(var/mob/user)
+	var/message
+	if(SUPER_FART in user.mutations)
```

Comment:
Ah right, I thought it had been overlooked when super farts were brought in. Will strip it out